casper/src/main/scala/coop/rchain/casper/util/comm/CommUtil.scala

```diff
@@ -9,26 +9,28 @@ import coop.rchain.casper.protocol._
 import coop.rchain.comm.PeerNode
 import coop.rchain.comm.protocol.rchain.Packet
 import coop.rchain.crypto.codec.Base16
-import coop.rchain.p2p.effects._
 import coop.rchain.comm.transport._, CommMessages._
 import coop.rchain.comm.discovery._
 import coop.rchain.comm.CommError.ErrorHandler
 import coop.rchain.shared._
 
 import scala.util.Try
+import scala.concurrent.duration._
 
 object CommUtil {
 
   private implicit val logSource: LogSource = LogSource(this.getClass)
 
+  private val defaultTimeout = 1000.milliseconds
```

Comment:
I'm not a fan of this hardcoded timeout, would definitely prefer it to be configurable. And perhaps it makes more sense for it to belong in `TransportLayer` itself and have it be overridable at the call-site